Index: xserver-xorg-video-qxl/src/qxl_drmmode.c
===================================================================
--- xserver-xorg-video-qxl.orig/src/qxl_drmmode.c
+++ xserver-xorg-video-qxl/src/qxl_drmmode.c
@@ -37,12 +37,7 @@
 #include "X11/Xatom.h"
 #include "xf86DDC.h"
 /* DPMS */
-#ifdef HAVE_XEXTPROTO_71
 #include <X11/extensions/dpmsconst.h>
-#else
-#define DPMS_SERVER
-#include <X11/extensions/dpms.h>
-#endif
 #include <cursorstr.h>
 
 #include "qxl.h"
